elastic/kibana
# [Fleet] Fix secrets with dot.separated variable names

## Summary

Closes https://github.com/elastic/kibana/issues/173048

Move away from dot-separated secret paths in favor of array-based paths to fix the above bug around dot-separated variable names. 

## To test

Create an integration policy for the Universal Profiling Agent on v8.12.0 and observe that no errors occur during policy creation.
